src/main/java/com/extremelyd1/game/team/TeamFactory.java

```diff
@@ -16,44 +16,44 @@
      * The names of the teams
      */
     private final String[] names = {
-            "Red",
+            "Aqua",
```

Comment:
There is a specific reason for not putting these team names (and colors as well) in alphabetical order. I wanted the first teams that the TeamFactory uses to be teams that can be easily told apart. This is helpful for people that have issues distinguishing colors.

Having said that, I also think that the auxiliary teams you have added should thus be at the bottom of this array (both the names and colors).